src/com/lftechnology/java/training/srijan/basics/football/ManchesterUnited.java

```diff
@@ -0,0 +1,70 @@
+package com.lftechnology.java.training.srijan.basics.football;
+
+import java.util.logging.Logger;
+import java.util.InputMismatchException;
+import java.util.NoSuchElementException;
+import java.util.Scanner;
+
+public class ManchesterUnited {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Add a private constructor to hide the implicit public one. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1118)